examples/gno.land/p/demo/uassert/uassert_test.gno

```diff
@@ -218,8 +218,8 @@ func TestEmpty(t *testing.T) {
 		{"", true},
 		{0, true},
 		{int(0), true},
-		{int64(0), true},
-		{uint(0), true},
+		{int64(0), false},
+		{uint(0), false},
 		// XXX: continue
```

Comment:
the original assertion is incorrect, see:
https://go.dev/play/p/B1PBlaB96Gd

Reply:
Nice. Can you fix `uassert` to does what is expected? (return true even for int64(0)?)

A bit out of scope, but I think it's reasonable.

Reply:
the is fixed.